python-avd/pyavd/_cv/client/models.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from typing import Literal, NamedTuple, Self
+
+from pyavd._cv.api.arista.tag.v2 import ElementType, Tag, TagAssignment
+
+ELEMENT_TYPE_MAP = {
+    "device": ElementType.DEVICE,
+    "interface": ElementType.INTERFACE,
+    "unspecified": ElementType.UNSPECIFIED,
+}
+
+REVERSED_ELEMENT_TYPE_MAP = {
+    ElementType.DEVICE: "device",
+    ElementType.INTERFACE: "interface",
+    ElementType.UNSPECIFIED: "unspecified",
+}
+
+
+class TagTuple(NamedTuple):
+    """
+    Represent the input model for a CloudVision Tag.
+
+    Attributes:
+        element_type: The type of network element the tag applies to ('device' or 'interface').
+        label: The label of the tag.
+        value: The value of the tag.
+    """
+
+    element_type: Literal["device", "interface", "unspecified"]
+    label: str
+    value: str
+
+    @property
+    def element_type_member(self) -> ElementType:
```

Comment:
```suggestion
    def as_element_type(self) -> ElementType:
```
Removed property on purpose.